msg_helpers.go

```diff
@@ -445,6 +445,14 @@ Option:
 		}
 		edns = append(edns, e)
 		off += int(optlen)
+	case EDNS0EXPIRE:
+		e := new(EDNS0_EXPIRE)
+		e.Code = code
```

Comment:
why this assignment?

Reply:
Since EDNS0_EXPIRE struct has Code field, just fill it with. However, if it's not consistent with other cases, I can remove e.Code assignment. Please let me know.

Reply:
However, i think it's good idea to fill the code with correct value. Otherwise, I am seeing zero code value in final return message.

Reply:
I don't think this is appropriate. Maybe it would make sense, but none of the others set it even though they have the field too. Consistency is what should be aimed for here.

Reply:
Sure, removed code assignment, to be consistent with other cases.